src/SessionSetup/RetrieveInstancesWidget.cpp

```diff
@@ -89,15 +89,15 @@ void RetrieveInstancesWidget::Start() {
   InitialLoad(LoadLastSelectedProjectFromPersistentStorage());
 }
 
-InstanceListScope RetrieveInstancesWidget::GetInstanceListScope() const {
+InstanceListScope RetrieveInstancesWidget::GetSelectedInstancesScope() const {
```

Comment:
The name is a bit confusing to me. I assume it's not about selected instances, right? It's about the selected instance-scope, isn't it? For whatever reason I immediately think about selected instances and I'm worried that others will also be confused.

What about GetSelectedInstanceListScope? Longer I know, but clearer I think.

Reply:
done